dom0-updates/qubes-dom0-update

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
-UPDATEVM=`qubes-prefs --get updatevm`
-UPDATES_STAT_FILE=/var/lib/qubes/updates/dom0-updates-available
+[ -n "$UPDATEVM" ] || UPDATEVM=`qubes-prefs --get updatevm`
```

Comment:
Retrieved global updatevm setting (you can change that using `qubes-prefs`). Do you think it would be useful to temporary change updatevm for just one `qubes-dom0-update` call?